Makefile

```diff
@@ -10,22 +10,55 @@ CCP_PATRONI_VERSION ?= 1.6.5
 CCP_BACKREST_VERSION ?= 2.27
 CCP_VERSION ?= 4.4.0
 CCP_POSTGIS_VERSION ?= 3.0
+PACKAGER ?= yum
 
 # Valid values: buildah (default), docker
 IMGBUILDER ?= buildah
+# Determines whether or not images should be pushed to the local docker daemon when building with
+# a tool other than docker (e.g. when building with buildah)
+IMG_PUSH_TO_DOCKER_DAEMON ?= true
 IMGCMDSTEM=sudo --preserve-env buildah bud --layers $(SQUASH)
 DFSET=$(CCP_BASEOS)
+DOCKERBASEREGISTRY=registry.access.redhat.com/
+
+# Default the buildah format to docker to ensure it is possible to pull the images from a docker
+# repository using docker (otherwise the images may not be recognized)
+export BUILDAH_FORMAT ?= docker
 
 # Allows simplification of IMGBUILDER switching
 ifeq ("$(IMGBUILDER)","docker")
 	IMGCMDSTEM=docker build
 endif
 
 # Allows consolidation of ubi/rhel Dockerfile sets
+ifeq ("$(CCP_BASEOS)", "rhel7")
+        DFSET=rhel
+endif
+
 ifeq ("$(CCP_BASEOS)", "ubi7")
-	DFSET=rhel7
+        DFSET=rhel
+endif
+
+ifeq ("$(CCP_BASEOS)", "rhel8")
```

Comment:
Do we need to support this?

Reply:
I was not sure if we would have to support it or not.  I did not add any of the surrounding code in the Dockerfiles to do so, but left it in just in case.  If you want me to remove it just let me know

Reply:
Yeah, let's remove it. If we don' thave it any any of the Dockerfiles, I don't see why we need it in the Makefile :smile: